src/view/reviewDocumentCommentProvider.ts

```diff
@@ -718,6 +726,57 @@ export class ReviewDocumentCommentProvider implements vscode.DocumentCommentProv
 		});
 	}
 
+	public async addReaction(document: vscode.TextDocument, comment: vscode.Comment, reaction: vscode.CommentReaction) {
+		await this.editReaction(document, comment, reaction, true);
+	}
+
+	public async deleteReaction(document: vscode.TextDocument, comment: vscode.Comment, reaction: vscode.CommentReaction) {
```

Comment:
I keep getting `Deleting the comment reaction failed: Illegal argument: line must be non-negative.` when trying to remove a comment while a PR is checked out

Reply:
I digged into this bug and turns out to be not related to this pr. It's triggerred by the comments update service, I'll work on another PR to fix this.